packages/expo-ui/ios/Modifiers/ViewModifierRegistry.swift

```diff
@@ -826,6 +826,49 @@ internal struct ContainerShapeModifier: ViewModifier, Record {
   }
 }
 
+internal struct ButtonStyleModifier: ViewModifier, Record {
+  @Field var style: String = "automatic"
```

Comment:
maybe use enum than string?

Reply:
Thanks for catching. [Done](https://github.com/expo/expo/pull/40119/commits/2b9bcdeb22c9b484bf467e535df7ba5acddf0ad6)